creator-node/src/middlewares/premiumContent/premiumContentMiddleware.ts

```diff
@@ -73,38 +79,51 @@ export const premiumContentMiddleware = async (
 
     switch (error) {
       case PremiumContentAccessError.MISSING_HEADERS:
-        return sendResponse(
+        prometheusResult.result = 'abort_premium_content_missing_headers'
+        return sendResponseWithMetric(
           req,
           res,
           errorResponseUnauthorized(
             'Missing request headers for premium content.'
-          )
+          ),
+          prometheusResult
```

Comment:
i see why you did it this way but might be cleaner if you just specified the object in each `sendResponseWithMetric` call instead of setting a default `prometheusResult` at the top and then overriding it each time - just more, confusing code this way

Reply:
👍 fair. making update